src/components/AddressResolver/AddressResolver.tsx

```diff
@@ -1,36 +1,38 @@
 import React, { FunctionComponent, useState } from "react";
-import { Plus } from "react-feather";
 import { Button } from "../UI/Button";
 import { AddressesTable } from "./AddressesTable";
+import { IconAddFile } from "../UI/Icon";
 
 export const AddressResolver: FunctionComponent = () => {
   const [isNewEndpoint, setNewEndpoint] = useState(false);
 
   return (
     <>
-      <div className="flex items-end">
+      <div className="flex flex-col md:flex-row md:items-end">
         <div className="flex-1 flex-col">
-          <h3>Settings: Address Resolver</h3>
-          <p className="mb-0 text-gray-500">Add third party’s endpoint to resolve addresses. </p>
+          <h3 className="font-ubuntu text-4xl text-cloud-900">Resolver: Address</h3>
+          <p className="mb-5 text-cloud-900">Add third party’s endpoint to resolve addresses.</p>
         </div>
```

Comment:
omit this and leave it at application end (tt-web, creator)

Reply:
done.